dev.paullee.scraterpreter.Scrapec.json

```diff
@@ -0,0 +1,45 @@
+{
+    "app-id": "dev.paullee.scraterpreter.Scrapec",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "scrapec",
+    "finish-args": [
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "scrapec",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app --no-deps ."
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/scraterpreter/scrapec.git",
+                    "tag": "20.06.3",
+                    "commit": "e2ffcd545ab6aff22f7f067803db940dcf77c9dd"
+                },
+                {
+                    "type": "file",
+                    "path": "dev.paullee.scraterpreter.Scrapec.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "dev.paullee.scraterpreter.Scrapec.metainfo.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "dev.paullee.scraterpreter.Scrapec.png"
+                }
+            ],
+            "builddir": true,
+            "post-install": [
+                "install -Dm644 dev.paullee.scraterpreter.Scrapec.png /app/share/icons/hicolor/128x128/apps/dev.paullee.scraterpreter.Scrapec.png",
```

Comment:
You don't need an icon nor a desktop file for a console application

Reply:
Oh, okay. But then how do I set the thumbnail on the flathub?

Reply:
The console applications are not displayed on the Flathub frontend yet

Reply:
Oh. So should I remove the icon or keep it in to keep the thumbnail for when console applications start being featured on the Flathub?

Reply:
Anyhow I have deleted the icons and the desktop file. Thanks